OTKit/otkit-ticketing-desktop/token.yml

```diff
@@ -0,0 +1,27 @@
+props:
+  # Used in ticketing modal of desktop restaurant profile
+  # =============================================
+  modal-margin-vertical:
+    type: "size"
+    value: "48px"
+
+  # Used in ticketing modal of desktop restaurant profile
+  # =============================================
+  modal-max-width:
+    type: "size"
+    value: "672px"
+
+  # Used in ticketing modal of desktop restaurant profile
+  # =============================================
+  modal-background-color:
+    value: "rgba(0,0,0,0.8)"
+
+  # Used in ticketing modal of desktop restaurant profile
+  # =============================================
+  modal-hero-fallback-background-color:
+    value: "#492c6e"
```

Comment:
I don't think @francesyun approved this, but I added a shade of purple that is close to the hamster ball hero image. Feel free to change @francesyun 😭

Reply:
haha, this is fine with me :)